include/seqan3/std/concepts

```diff
@@ -51,14 +52,62 @@
  *
  */
 
-namespace std
-{
-
 /*!\defgroup concepts concepts
  * \ingroup std
  * \brief The \<concepts\> header from C++20's standard library.
  */
 
+#if 0
```

Comment:
What was the rationale?
I vaguely remember that there was a reason...

Reply:
The boolean concept is currently broken, see the issues linked above.

Reply:
Ok so this should link to the issue and should be activated once the issue is fixed?